scripts/run-in-pipeline.sh

```diff
@@ -0,0 +1,200 @@
+#   Output:
+#     - case-insentive value (true/false) or default value
+##
+function normalize {
+    local user_param=$1
+    local default_val=$2
+
+    if [[ "${user_param,,}" == "$_true" ]]  # Note:  ${var,,} means to lowercase of var
+    then
+        result="$_true"
+    elif [[ "${user_param,,}" == "$_false" ]]
+    then
+        result="$_false"
+    else
+        result="$default_val"
+    fi
+
+    echo $result
+}
+
+## Initialize Detect Secrets audit_report parameter string
+audit_report_params=' '
+
+##
+# Starting the pipeline Detect Secrets run
+##
+echo "[ Starting Detect Secrets run ]"
+echo
+echo "...using Baseline: $baseline"
+
+##
+# Checking Env Vars and appending coresponding Detect Secrets audit report parameters to the parameter string
+# Note:  Env Var input values are normalized before being checked. Thus case-insensitive and/or defaulted if necessary
+#        Thus the values will always be either true or false after they are normalized
+##
+
+# skip_scan parameter
+skip_scan="$(normalize $skip_scan $_skip_scan_default)"
+if [[ "$skip_scan" == "$_true" ]]
+then
+  echo "...skip scan with baseline update: $_true"
+else
+  echo "...skip scan with baseline update: $_false"
+fi
+
+# json parameter
+json="$(normalize $json $_json_default)"
+if [[ "$json" == "$_true" ]]
+then
+  echo "...output json: $_true"
+  audit_report_params="$audit_report_params $_json_option"
+else
+  echo "...output json: $_false"
+fi
+
+# omit_instructions parameter
+omit_instructions="$(normalize $omit_instructions $_omit_instructions_default)"
+if [[ "$omit_instructions" == "$_true" ]]
+then
+  echo "...omit instructions: $_true"
+  audit_report_params="$audit_report_params $_omit_instructions_option"
+else
+  echo "...omit instructions: $_false"
+fi
+
+
+# Fail On Live
+fail_live="$(normalize $fail_live $_fail_live_default)"
+if [[ "$fail_live" == "$_true" ]]
+then
+  echo "...fail on live: $_true"
+  audit_report_params="$audit_report_params $_fail_live_option"
+else
+  echo "...fail on live: $_false"
+fi
+
+# Fail On Unaudited
+fail_unaudited="$(normalize $fail_unaudited $_fail_unaudited_default)"
+if [[ "$fail_unaudited" == "$_true" ]]
+then
+  echo "...fail on unaudited: $_true"
+  audit_report_params="$audit_report_params $_fail_unaudited_option"
+else
+  echo "...fail on unaudited: $_false"
+fi
+# Fail On Audited Real
+fail_audited_real="$(normalize $fail_audited_real $_fail_audited_real_default)"
+if [[ "$fail_audited_real" == "$_true" ]]
+then
+  echo "...fail on audited real: $_true"
+  audit_report_params="$audit_report_params $_fail_audited_real_option"
+else
+  echo "...fail on audited real: $_false"
+fi
+
+## Calling Detect Secrets scan with baseline update to create or update existing baseline file
+if [[ "$skip_scan" == "$_false" ]]
+then
+  echo
+  echo "Scanning code directory (docker volume mounted to $PWD) and updating baseline file $baseline ... "
+  echo
+  detect-secrets scan --update $baseline
+fi
+
```

Comment:
I realized that I suggested adding an echo in the wrong place. If `SKIP_SCAN` is true, then the spacing isn't consistent:
```
$ docker run  --env SKIP_SCAN=True -it -a stdout --rm -v $(pwd):/code git-defenders/detect-secrets-ubi
[ Starting Detect Secrets run ]

...using Baseline: .secrets.baseline
...skip scan with baseline update: true
...output json: false
...omit instructions: false
...fail on live: true
...fail on unaudited: true
...fail on audited real: true
Running report: Baseline .secrets.baseline - Options:  --fail-on-live --fail-on-unaudited --fail-on-audited-real


10 potential secrets in .secrets.baseline were reviewed. All checks have passed.

        - No unaudited secrets were found

        - No live secrets were found

        - No secrets that were audited as real were found


[ Ending Detect Secrets - run succeeded ]
```

```suggestion
  detect-secrets scan --update $baseline
fi
 
echo

```

This change results in this:
```
$ docker run  --env SKIP_SCAN=True -it -a stdout --rm -v $(pwd):/code git-defenders/detect-secrets-ubi
[ Starting Detect Secrets run ]

...using Baseline: .secrets.baseline
...skip scan with baseline update: true
...output json: false
...omit instructions: false
...fail on live: true
...fail on unaudited: true
...fail on audited real: true

Running report: Baseline .secrets.baseline - Options:  --fail-on-live --fail-on-unaudited --fail-on-audited-real


10 potential secrets in .secrets.baseline were reviewed. All checks have passed.

        - No unaudited secrets were found

        - No live secrets were found

        - No secrets that were audited as real were found


[ Ending Detect Secrets - run succeeded ]
```

and the spacing is still consistent when `SKIP_SCAN` is false.